.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: "Build"
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # 3.3.0
+        with:
+          fetch-depth: 0
```

Comment:
Any specific reason for fetching all history?

Reply:
Nope, just copypasta. I'll change it.